js/main.js

```diff
@@ -19,8 +19,130 @@ function formSubmit(event) {
   objform.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(objform);
+  var entrytest = renderJournalEntry(objform);
+  ul.prepend(entrytest);
+  if (data.entries.length > 0) {
```

Comment:
You don't need this conditional. If a new entry has been submitted, it is guaranteed that you have more than zero entries, so you can always hide this element